tests/test_ctf/robots_txt/chal.yaml

```diff
@@ -2,4 +2,4 @@
   name: Robots.txt
   flag: flag{this_is_a_test}
   config:
-    asdf: asdf
```

Comment:
what's the asdf for?

Reply:
No idea. This was just in the chal.yaml file when I downloaded chalgen. It seemed it was just a placeholder as it wasn't being called anywhere in web.py, so I just removed it and replaced it with the prompt option in the config.

Reply:
oh you removed it, nice. I thought this was an addition from looking at it on mobile